### ISSUE TITLE
Excluding Decoder Network!

### ISSUE THREAD

**Issue by iphton** [XifengGuo/CapsNet-Keras#89]
@XifengGuo Hello, If we want to exclude the decoder part of CapsNet, how can we get the probability of `out_caps` - the output of the `Length` function? Should we need to apply simply a softmax classifier or it's already given?

```python

def CapsNet(input_shape, n_class, routings):
    x = layers.Input(shape=input_shape)
                                                                                                                            
    # Layer 1: Just a conventional Conv2D layer
    conv1 = 

    # Layer 2:
    primarycaps = PrimaryCap(conv)

    # Layer 3: Capsule layer. Routing algorithm works here.
    digitcaps = CapsuleLayer()(primarycaps)
    
    # Layer 4: 
    out_caps = Length(name='capsnet')(digitcaps)

    # Models for training 
    #train_model = models.Model([x, y], [out_caps, decoder(masked)]) # encoder + decoder
    train_model = models.Model(inputs=[x], outputs=[out_caps])  # only encoder

    return train_model

```
